content/applications/finance/accounting/bank/reconciliation/bank_reconciliation.rst

```diff
@@ -0,0 +1,86 @@
+===========================
+Reconciling bank statements
```

Comment:
```suggestion
Reconciling bank statements
```
We usually avoid 'verb+ing' headings. Maybe simply 'Bank statements'? Since it's under the 'Bank reconciliation' page, it's maybe not needed to mention reconciliation again.